client/my-sites/site-logs/main.tsx

```diff
@@ -16,14 +17,24 @@ export function SiteLogs() {
 	const [ startTime ] = useState( moment().subtract( 7, 'd' ).unix() );
 	const [ endTime ] = useState( moment().unix() );
 
-	const { data } = useSiteLogsQuery( siteId, {
-		logType: 'web',
+	const [ logType, setLogType ] = useState< 'php' | 'web' >( 'php' );
```

Comment:
The query uses this `logType` state as the source of truth. The tab panel uses the query parameter as the source of truth. But I think they should both be using the same source of truth.

One option would be to have both the query and tab panel use the `logType` state, and the query param is only used on initial render as a way to look up what the initial selection should be.

```suggestion
	const [ logType, setLogType ] = useState< 'php' | 'web' >( () => {
		const queryParam = new URL( window.location.href ).searchParams.get( 'logType' );
		return [ 'php', 'web' ].includes( queryParam ) ? queryParam : 'php';
	 } );
```

Reply:
yeah this make sense.